optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/java/org/optaplanner/workbench/screens/solver/backend/server/ToSolverConfigModel.java

```diff
@@ -85,6 +93,21 @@ private TerminationConfigModel create( final TerminationConfig terminationConfig
             model.setUnimprovedSecondsSpentLimit( terminationConfig.getUnimprovedSecondsSpentLimit() );
             model.setUnimprovedMillisecondsSpentLimit( terminationConfig.getUnimprovedMillisecondsSpentLimit() );
 
+            model.setBestScoreLimit( terminationConfig.getBestScoreLimit() );
+            model.setBestScoreFeasible( terminationConfig.getBestScoreFeasible() );
+
+            model.setStepCountLimit( terminationConfig.getStepCountLimit() );
+            model.setUnimprovedStepCountLimit( terminationConfig.getUnimprovedStepCountLimit() );
+
+            model.setCalculateCountLimit( terminationConfig.getCalculateCountLimit() );
```

Comment:
getCalculateCountLimit() -> getScoreCalculateCountLimit() (and similarly in other places)